runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -652,7 +660,18 @@ function LanguageTree:_get_injection(match, metadata)
 
     -- Lang should override any other language tag
     if name == 'injection.language' then
-      lang = vim.treesitter.get_node_text(node, self._source, { metadata = metadata[id] })
+      local text =
+        vim.treesitter.get_node_text(node, self._source, { metadata = metadata[id] }):lower()
+      if not text then
+        break
```

Comment:
I don't think this possible. `get_node_text()` returns `string` in all paths.

The case it doesn't is when `metadata.text` is defined but isn't a string.